src/create_atoms.cpp

```diff
@@ -319,11 +319,7 @@ void CreateAtoms::command(int narg, char **arg)
     memory->create(xmol, onemol->natoms, 3, "create_atoms:xmol");
   }
 
-  if (style == MESH) {
-    if (mode == MOLECULE)
-      error->all(FLERR, "Create_atoms mesh is not compatible with the 'mol' option");
-    if (scaleflag) error->all(FLERR, "Create_atoms mesh must use 'units box' option");
-  }
+  if (style == MESH && scaleflag) error->all(FLERR, "Create_atoms mesh must use 'units box' option");
```

Comment:
Using the "mesh" option with molecule files makes no sense. These changes need to be reverted.

Reply:
see my comment on slack.

Reply:
done. i copied my changed file outside local repo, downloaded file from develop branch, committed that, moved my modified file back in local repo and stashed that